examples/protobufjs/readme.md

```diff
@@ -0,0 +1,5 @@
+# protobufjs fuzz test
+
+This fuzz test uses
+[ECMAScript Modules](https://jestjs.io/docs/ecmascript-modules) and requires
+Node.js 18.8.0+.
```

Comment:
Why do we need such a new node for this? My not-even-6-months-old system comes with node node `v16.20.0`. According do the docs, ES Modules are stable in node since version `v12`.

(It's just an example project, so probably still fine …)